internal/cmd/gocdk/biome.go

```diff
@@ -246,17 +263,98 @@ func biomeApply(ctx context.Context, pctx *processContext, biome string, input b
 		return xerrors.Errorf("biome apply %s: %w", biome, err)
 	}
 
-	c := pctx.NewCommand(ctx, biomePath, "terraform", "init", "-input="+strconv.FormatBool(input))
-	if err := c.Run(); err != nil {
+	writeTerraformOutput := func(cmd *exec.Cmd, msg []byte) {
+		const separator = "#########################################\n"
+		pctx.stderr.Write([]byte("\n"))
+		pctx.stderr.Write([]byte(separator))
+		pctx.stderr.Write([]byte(fmt.Sprintf("Output from %q\n", strings.Join(cmd.Args, " "))))
+		pctx.stderr.Write([]byte(separator))
+		pctx.stderr.Write(msg)
+		pctx.stderr.Write([]byte(separator))
+		pctx.stderr.Write([]byte("\n"))
+	}
+
+	pctx.Logf("Checking to see if resource updates are needed...")
+
+	// First, run "terraform init". It's needed the first time, and anytime
+	// a new Terraform provider is added, so it's safest to just always run it.
+	inputArg := fmt.Sprintf("-input=%v", opts.Input)
+	c := pctx.NewCommand(ctx, biomePath, "terraform", "init", inputArg)
+	c.Env = overrideEnv(c.Env, "TF_IN_AUTOMATION=1")
+	c.Stdout, c.Stderr = nil, nil
+	if out, err := c.CombinedOutput(); err != nil {
+		writeTerraformOutput(c, out)
+		return xerrors.Errorf("biome apply %s: %w", biome, err)
+	} else if opts.Verbose {
+		writeTerraformOutput(c, out)
+	}
+
+	// Next, run "terraform plan", writing the plan to a temp file.
+	planFile, err := ioutil.TempFile(pctx.workdir, "tfplan")
```

Comment:
Just checking: is this intentionally writing the file in the current directory rather than the system temporary directory? I don't see a good reason to expose the file to the user unless the idea is to permit the user to re-execute the plan in case of failure. However, that seems to not be the case, since below there is a `defer os.Remove(...)`.

Reply:
Yes, it is intentional, although not for a great reason. See the comment a couple of lines down (maybe I should move it up?).

```
	// Note: using just the base filename instead of passing the file path to --out
	// because Terraform appears to not work with full Windows paths.
```

Basically, I can't get `terraform apply` to work on Windows when passed a full path to a plan file. It just exits with status code 1 and no message. It works fine when the plan file is in the current directory and you pass it just the filename.

Reply:
Tried to improve the comments for this.